src/js/shot-generator/Editor.js

```diff
@@ -1809,13 +1909,44 @@ const InspectedElement = ({ sceneObject, models, updateObject, selectedBone, mac
         positionSliders
       ],
 
-      sceneObject.type == 'object' && [
+      (sceneObject.type == 'object' ) && [
         [
           'div.column',
           volumeSliders
         ],
       ],
 
+      sceneObject.type == 'volume' && [        
+        [
+          'div.column',
+          // [VolumePresetsEditor2, { sceneObject }],
+          [VolumePresetsEditor, { sceneObject }],
+          [NumberSlider, { label: 'width', value: sceneObject.width, min: 0.1, max: 25, onSetValue: createOnSetValue(sceneObject.id, 'width') } ],
+          [NumberSlider, { label: 'height', value: sceneObject.height, min: -25, max: 25, onSetValue: createOnSetValue(sceneObject.id, 'height') } ],
+          [NumberSlider, { label: 'depth', value: sceneObject.depth, min: 0.1, max: 25, onSetValue: createOnSetValue(sceneObject.id, 'depth') } ], 
+          [NumberSlider, { 
```

Comment:
This is the one i didn't manage to set up, so right now i'm just converting the values i get to int